about.html

```diff
@@ -1,73 +1,166 @@
+
+            </div>
+            <img src="img/jumbo-about.png" alt="jumboRoad">
+            <div>
+
+            </div>
+
+
+        </header>
+
+        <!-- End Nav-Content Top -->
+
+
+        <!-- About Section -->
+
+        <section>
+
+            <div class="aboutsection">
+
+                <h1>About</h1>
+
+                <p> Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view
+                    of disruptive innovation via workplace diversity and empowerment.</p>
+
+            </div>
+
+        </section>
+
+        <!-- End About Section -->
+
+        <!-- Image Group 1 -->
+
+        <section>
+
+
+
+
+            <div class="boxmain">
+
 
 
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+                <div class="leftbox">
 
-    Learn More
+                    <img src="img/about-plan.png" alt="strategy">
 
+                    <h2>Strategy</h2>
 
+                    <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+                    </p>
 
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+                    <button>Learn More</button>
 
-    Learn More
+                </div>
 
 
 
-    <img src="img/about-office.png" alt="strategy">
-    
-    Places We Work
+                <div class="rightbox">
 
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+                    <img src="img/about-working.png" alt="strategy">
 
-    Learn More
+                    <h2>How We Work</h2>
 
-    <img src="img/about-meeting.png" alt="strategy">
-    
-    Collaboration
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+                    <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+                    </p>
 
-    Learn More
+                    <button>Learn More</button>
 
-    Let's Work Together
+                </div>
 
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-        
-      <footer>
-        <nav>
-          <a href="index.html">Home</a>
-          <a href="#">About</a>
```

Comment:
always link to the html if you can, even if its the about page and your on the about page! :)